server.js

```diff
@@ -38,4 +43,5 @@ webhook.on('*', event => {
   }
 });
 
+debug('Listening on http://localhost:' + PORT);
 console.log('Listening on http://localhost:' + PORT);
```

Comment:
Since this should aways be output, it's probably fine to just do the `console.log`.